### DISCUSSION TITLE
uBO spams the logger with scriptlet filter

### DISCUSSION THREAD

**Discussion by MasterKia** [uBlockOrigin/uBlock-issues#2537] [Q&A] [answered]
Add to "My filters":
```adb
gamesetup.ir##+js(acs, vc_url)
gamesetup.ir##+js(set, random, false)
```

Open the logger.

Visit and wait for it to load completely:
`https://gamesetup.ir/`

See huge amounts of `##+js` logs:
![Screenshot_20230311_201326](https://user-images.githubusercontent.com/17685483/224496836-c1cbeed3-2f85-4612-ad11-551a02b15bf3.png)

Is this intended behavior?

Edit:
1.47.5b6 dev
Firefox 110

**Comment by ghost** [comment, marked as answer]
Dev version?

**Reply by mapx-** [reply]
getting same behaviour in chrome, uBO 1.47.5b4

**Reply by MasterKia** [reply]
Sorry, forgot to mention.
1.47.5b6 dev
Firefox 110

**Reply by gorhill** [reply]
The page is just creating a lot of browsing contexts and as a result content scripts are being injected for all of these created contexts. uBO is just reporting what it is doing. Open console and type `document.querySelectorAll('object')`, I get 68 `object` each with a URL such as `https://gamesetup.ir/wp-content/themes/lw-gamesetup/images/icon-folder.svg`. The browser creates a browser context for each `object`, so all is working as expected.
